### ISSUE TITLE
Telegram Notification

### ISSUE THREAD

**Issue by jrodriguesmoreit** [louislam/uptime-kuma#3561]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Hi.
Is it possible to send in addition to the Friendly Name, send the group?
Example Group-Friendly Name?
Thanks

### 📝 Error Message(s) or Log

Send telegram notification only Friendly Name

### 🐻 Uptime-Kuma Version

1.22.1

### 💻 Operating System and Arch

 Ubuntu 22.04.2 LTS

### 🌐 Browser

Chrome

**Comment by CommanderStorm** [comment]
Notification templating is currently not supported.
See #2617 and other duplicates of https://github.com/louislam/uptime-kuma/issues/975

**Comment by CommanderStorm** [comment]
To resolve this, you can open a `feature-request` or an [empty draft PR](https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma)

**Comment by jrodriguesmoreit** [comment]
Tks.
Its possible change the code for that?
I change the line 1221 let msg = `[${monitor.name}] [${text}] ${bean.msg}`; in uptime-kuma/server/model/monitor.js but without any efect.
Which line should I change?
Thanks

**Comment by CommanderStorm** [comment]
> Its possible change the code for that?

Notification templating is planned, but will likely require a reattempt of https://github.com/louislam/uptime-kuma/pull/760

As stated above, to change the current naming-behaviour, please open an [empty draft PR](https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma) first or open a `feature-request` instead.

> without any efect

What were your steps? (Have you followed the contributing guide or the deployment guide?)

**Comment by bernarddt** [comment]
Just to expand on this requests information. I've also now made extensive use of the groups, it's awesome n the GUI! I used to use Uptime Robot that don't have this feature and it was a bit chaotic! It allows me to group my servers and services on them like:

Server Name > SQL > Server HostName 
Server Name > ASP > Server HostName

The reference in Telegram does not indicate the group yet, see how it is currently:

![2023-08-18 10_38_09-Window](https://github.com/louislam/uptime-kuma/assets/151132/517c4951-f71e-44d4-8c02-509c9abf023a)
